CONTRIBUTING.md

```diff
@@ -23,3 +23,29 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
```

Comment:
i would use literally anything other than `small`. 2 reasons:

- downplays the effort of new contributors
- doesn't properly signal that these issues should be reserved for new contributors

recommendations:  `your-first-PR`, `starter-issue`

i would also recommend a `PRs welcome` tag that flags that PRs are welcome, and then put another tag alongside it that flags that it would be good for a new contributor

another consideration: when you talk about "first" issues, do you mean first contribution to the rspec collection of software, OR first contribution ever? this will inform the name of the tag you use as well as the amount of scaffolding that you'll need to provide in the issue.